pkg/render/intrusion_detection.go

```diff
@@ -1760,6 +1810,26 @@ func (c *intrusionDetectionComponent) getBaseADDetectorsPodTemplate(podTemplateN
 			Name:  "FIPS_MODE_ENABLED",
 			Value: operatorv1.IsFIPSModeEnabledString(c.cfg.Installation.FIPSMode),
 		},
+		{
+			Name:  "LINSEED_URL",
+			Value: relasticsearch.LinseedEndpoint(c.SupportedOSType(), c.cfg.ClusterDomain),
+		},
+		{
+			Name:  "LINSEED_CA",
```

Comment:
I think the variable names could be improved upon:
LINSEED_CA -> LINSEED_CERT (We happen to mount a CA bundle, but it is used to validate Linseed's cert)
LINSEED_CLIENT_CERT -> TLS_CRT (This is not specific to linseed, it is the tls keypair of intrusion detection pods)
LINSEED_CLIENT_KEY -> TLS_KEY

Reply:
I understand, but I would rather keep it consistent across components. These are the names we have been using in our configurations for all components and in the Linseed client.